### ISSUE TITLE
.inf not parsed correctly in math syntax

### ISSUE THREAD

**Issue by FLomb** [calliope-project/calliope#613]
### What happened?

If using `.inf` in the math syntax in yaml files (e.g., `- expression: sum(cost, over=[nodes, techs]) <= .inf`), this is apparently not correctly parsed. It appears as `None` in the backend.

### Which operating systems have you used?

- [ ] macOS
- [ ] Windows
- [X] Linux

### Version

v0.7.0dev3

**Comment by brynpickering** [comment]
Yes, pyomo considers `.inf` as there being no upper bound. We might pass it .inf but it stores it as None. If there was a problem with parsing, the parser would complain. This _shouldn't_ be an issue when solving the model as it's pyomo's decision to store it as None and it handles sending it to the solver just fine (in your case, it just wouldn't send it at all - it's a constraint that isn't constraining anything).

**Comment by FLomb** [comment]
I see, ok, then maybe we can consider the issue closed?
